src/Toast.tsx

```diff
@@ -21,7 +22,7 @@ const ToastComponent = ({ header, body, type }: ToastProps) => {
         <Error
           onClose={() => setShow(false)}
           show={show}
-          delay={hideDelay}
+          delay={time ?? hideDelay}
```

Comment:
I've never seen the double `??`. Looked it up and it's my new favorite: the nullish coalescing operator. Sounds very sci-fi!